lms/envs/common.py

```diff
@@ -2294,3 +2297,15 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# Sets the maximum number of courses listed on the homepage
+# If set to None, all courses will be listed on the homepage
+HOMEPAGE_COURSE_COUNT = 9
+
+
+if FEATURES.get('ENABLE_COURSE_DISCOVERY'):
+    # if COURSE DISCOVERY feature is enabled, COURSES_ARE_BROWSABLE should be enabled as well
+    FEATURES['COURSES_ARE_BROWSABLE'] = True
+else:
+    # if COURSE DISCOVERY feature is disabled, all courses should be listed on the homepage
+    HOMEPAGE_COURSE_COUNT = None
```

Comment:
NIT: Should this be "Homepage_Course_Max" so that 'None' makes sense as what lists all courses?